src/components/App/App.js

```diff
@@ -13,10 +12,42 @@ export default class App extends Component {
     }
   }
   componentDidMount = () => {
-    this.setState({movies: movieData.movies});
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(response => {
+        if(!response.ok) {
+          this.setState({allMoviesError: response.status});
+        } else {
+          return response.json();
+        }
+      }
+    ).then(data => {
+      this.setState({movies: data.movies});
+    });
   }
```

Comment:
I like how you use different properties in the state depending on which fetch request it was